homeassistant/components/sensor/fritzbox_callmonitor.py

```diff
@@ -30,10 +35,18 @@
 
 INTERVAL_RECONNECT = 60
 
+# Return cached results if phonebook was downloaded less then this time ago.
+MIN_TIME_PHONEBOOK_UPDATE = datetime.timedelta(hours=6)
+
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
     vol.Optional(CONF_HOST, default=DEFAULT_HOST): cv.string,
     vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
+    vol.Optional(CONF_PASSWORD, default='admin'): cv.string,
+    vol.Optional(CONF_USERNAME, default=''): cv.string,
```

Comment:
Callmonitor without username and password doesn't work for me.
I think for the username default='admin' could be a better fit as it seems to be the default user.
On the other hand there should (hopefully) not be a general fritz box password so a user defined one could be marked as required:
vol.Required(CONF_PASSWORD, default=''): cv.string,